src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,49 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.Set;
+public class LottoTicket {
+    private final LottoNumbers lottoNumbers;
+    private final int bonusNumber;
+
+    public LottoTicket(LottoNumbers lottoNumbers) {
+        this.lottoNumbers = lottoNumbers;
+        this.bonusNumber = -1;
+    }
```

Comment:
점진적 생성자패턴을 이용하면 각 생성자에서 변수 주입하기에 매번 모두 관리해야하는 번거로움을 줄일 수 있을 것 같네요!

```suggestion
    public LottoTicket(LottoNumbers lottoNumbers) {
        this(lottoNumbers, -1);
    }
```